loopin.go

```diff
@@ -487,7 +496,7 @@ func (s *loopInSwap) sendUpdate(ctx context.Context) error {
 // execute starts/resumes the swap. It is a thin wrapper around executeSwap to
 // conveniently handle the error case.
 func (s *loopInSwap) execute(mainCtx context.Context,
-	cfg *executeConfig, height int32) error {
+	executorWg *sync.WaitGroup, cfg *executeConfig, height int32) error {
```

Comment:
We could completely rely on the exectur's wait group so the `loopInSwap.wg` could be removed altogether. And then maybe we could just rename `exectutorWg` to `wg` for simplicity.